tests/pytest/produce_output_test.py

```diff
@@ -286,3 +286,70 @@ def test_create_nexus_output_file_nest(config_tmpdir, output_tmpdir, PETALODIR,
      petalo_exe = PETALODIR + '/bin/petalo'
      command   = [petalo_exe, '-b', '-n', '1', init_path]
      p         = subprocess.run(command, check=True, env=my_env)
+
+
+@pytest.mark.order(4)
+def test_create_nexus_output_file_phantom(config_tmpdir, output_tmpdir, PETALODIR, base_name_phantom):
+
+     init_text = f"""
+/PhysicsList/RegisterPhysics G4EmStandardPhysics_option4
+/PhysicsList/RegisterPhysics G4DecayPhysics
+/PhysicsList/RegisterPhysics G4RadioactiveDecayPhysics
+/PhysicsList/RegisterPhysics PetaloPhysics
+/PhysicsList/RegisterPhysics G4StepLimiterPhysics
+
+### GEOMETRY
+/nexus/RegisterGeometry FullRingInfinity
+
+### GENERATOR
+/nexus/RegisterGenerator Back2backGammas
+
+### ACTIONS
+/nexus/RegisterRunAction DefaultRunAction
+/nexus/RegisterEventAction PetaloEventAction
+/nexus/RegisterTrackingAction DefaultTrackingAction
+
+/nexus/RegisterPersistencyManager PetaloPersistencyManager
+
+/nexus/RegisterMacro {config_tmpdir}/{base_name_phantom}.config.mac
+"""
+     init_path = os.path.join(config_tmpdir, base_name_phantom+'.init.mac')
+     init_file = open(init_path,'w')
+     init_file.write(init_text)
+     init_file.close()
+
+     config_text = f"""
+/run/verbose 1
+/event/verbose 0
+/tracking/verbose 0
+
+/process/em/verbose 0
+
+/Geometry/FullRingInfinity/depth 3. cm
+/Geometry/FullRingInfinity/sipm_pitch 7. mm
+/Geometry/FullRingInfinity/inner_radius 380. mm
+/Geometry/FullRingInfinity/sipm_rows 278
+/Geometry/FullRingInfinity/instrumented_faces 1
+/Geometry/FullRingInfinity/phantom true
+
+/Geometry/SiPMpet/efficiency 0.2
+/Geometry/SiPMpet/visibility true
+/Geometry/SiPMpet/time_binning 5. picosecond
+/Geometry/SiPMpet/size 6. mm
+
+/Generator/Back2back/region JPHANTOM
+
+/nexus/persistency/outputFile {output_tmpdir}/{base_name_phantom}
+/nexus/random_seed 16062020
+
+"""
+
+     config_path = os.path.join(config_tmpdir, base_name_phantom+'.config.mac')
+     config_file = open(config_path,'w')
+     config_file.write(config_text)
+     config_file.close()
+
+     my_env    = os.environ
+     petalo_exe = PETALODIR + '/bin/petalo'
+     command   = [petalo_exe, '-b', '-n', '10000', init_path]
```

Comment:
Last thing, I promise! :) Is it not enough with 100 evts?

Reply:
No problem, that's the purpose of a review :-). The larger the statistics, the better to check this, so I decided to go for > 1000, and it0s quite fast anyway, since this job doesn't generate optical photons.

Reply:
Ah, I didn't realized it does not generate optical photons. Then I approve!